syntax = "proto3";

package chatroom.message;

message UserLoginAck {
    bool success = 1;
    string errmsg = 2;
}